core/settings.py

```diff
@@ -216,6 +220,11 @@
         environment=ENV or "local",
     )
 
+# add debug toolbar when not running tests
+if (not TESTING) or (not DEBUG):
```

Comment:
I think this should be and instead of or

```suggestion
if (not TESTING) and (not DEBUG):
```

Which can be simplified to

```suggestion
if not (TESTING or DEBUG):
```

Reply:
I've swapped these for 
```py
if DEBUG and not TESTING:
```
to take into account the above request about making sure we're reliant on `DEBUG` being set.